File: algo/problems/0217-contains-duplicate/scala/ContainsDuplicateTest.scala
//> using scala 3.4.2
//> using test.dep org.scalameta::munit::1.0.0

class ContainsDuplicateTest extends munit.FunSuite:
  test("has dup") { assert(ContainsDuplicate.containsDuplicate(Array(1, 2, 3, 1))) }
  test("unique")  { assert(!ContainsDuplicate.containsDuplicate(Array(1, 2, 3, 4))) }
  test("empty")   { assert(!ContainsDuplicate.containsDuplicate(Array.empty[Int])) }

File: algo/problems/0217-contains-duplicate/js/containsDuplicate.test.js
const { containsDuplicate } = require('./containsDuplicate');

test('has dup', () => { expect(containsDuplicate([1, 2, 3, 1])).toBe(true); });
test('unique', () => { expect(containsDuplicate([1, 2, 3, 4])).toBe(false); });
test('empty', () => { expect(containsDuplicate([])).toBe(false); });
